tests/publiccloud/transfer_repos.pm

```diff
@@ -24,14 +25,15 @@ sub run {
     my $remote = $args->{my_instance}->username . '@' . $args->{my_instance}->public_ip;
     my @addons = split(/,/, get_var('SCC_ADDONS', ''));
     my $skip_mu = get_var('PUBLIC_CLOUD_SKIP_MU', 0);
-
+    my $repodir = "/tmp/repos/";
+    # VM repos.dir update,for limited-rw FS like micro
+    $repodir = "/opt/repos/" if (is_sle_micro);
```

Comment:
I vote for keeping this directory the same for all products.

Reply:
ok